### PR TITLE
Add lecture.rb

### PR DESCRIPTION
**Pull request by andy-mill**
Adding a new class Lecture. The Course class should have an array of lectures as its member.

### CODE REVIEW

**Comment by bayramkl** [comment]
This class should also have a field holding a AWS S3 key of a video in case it is a video lecture.

**Comment by bayramkl** [comment]
I guess there is typo `vidoe_key`

**Comment by andy-mill** [comment]
Fixed. 😊

File: lecture.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Lecture
+  attr_reader :title, :duration, :video_key
+
+  def initialize(attrs)
+    @title = attrs[:title]
+    @duration = attrs[:duration]
```
Comment (bayramkl) [inline lecture.rb:6]: Please convert duration to int just in case

Comment (bayramkl) [inline lecture.rb]: Please convert duration to int just in case `@duration = attrs[:duration].to_i`

**Pull request merged**